entries/t/twingate.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Twingate": {
+    "domain": "twingate.com",
+    "tfa": [
+      "totp",
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "Biometrics",
+      "Yubico OTP"
```

What's described in the documentation sounds a lot like FIDO2 U2F.
As a rule of thumb, if Windows Hello or "Security Keys" are mentioned, it is most likely U2F.
```suggestion
      "u2f"
```

---

I see. Also, do biometric features such as fingerprints not count? @Carlgo11